src/integration-test/java/io/aiven/kafka/connect/http/AvroIntegrationTest.java

```diff
@@ -78,7 +79,7 @@ public class AvroIntegrationTest {
 
     private static File pluginsDir;
 
-    private static final String DEFAULT_TAG = "5.4.3";
+    private static final String DEFAULT_TAG = "6.2.2";
```

Comment:
Perhaps decreasing this to the version that corresponds to 2.6.0 Kafka also would make sense..

Reply:
Done.